### PR TITLE
Adds skinny banner and teacher dashboard banner for Transformers

### PR DESCRIPTION
**Pull request by hannahbergam**
This PR adds both the skinny banner to the elementary page and the larger banner to the teacher and student dashboards. 

I used [this pr](https://github.com/code-dot-org/code-dot-org/pull/60450/files) as a guide for the teacher dashboard banner update.

Here are pics of the student and teacher dashboards, and the elementary page with and without the banner (english and non-english):

Screenshot 2024-08-30 at 9 17 27 AM

Screenshot 2024-08-30 at 9 17 39 AM

Screenshot 2024-08-30 at 9 18 46 AM

Screenshot 2024-08-29 at 5 01 04 PM

localhost code org_3000_student_elementary

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-2121

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/student/elementary.haml [hunk 1]
```diff
@@ -7,12 +7,14 @@ theme: responsive_full_width
 
 = view :poetry_banner
 
-%section.no-padding-bottom
+%section
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/student/elementary.haml]: Given the following section only appears in English, is there weird padding in non-EN? I haven't tested, just curious.

File: dashboard/config/marketing/announcements.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "pages": {
-    "/home": "user-research-aug-2024",
+    "/home": "transformers-sept-2024",
     "/student-home": ""
```
Comment (bethanyaconnor) [inline dashboard/config/marketing/announcements.json]: 100% a product question but do we not want to show this for students?

**Review by bethanyaconnor — APPROVED** [review]
FYI I think this will cause eyes diffs. Should be easy to approve though :)

File: dashboard/config/marketing/announcements.json (hunk 2 above)
Comment (dmcavoy) [reply dashboard/config/marketing/announcements.json]: Should be for both teachers and students.

File: pegasus/sites.v3/code.org/public/student/elementary.haml (hunk 1 above)
Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/student/elementary.haml]: I added a 3rem top margin to the banner instead. Thanks!

File: dashboard/config/marketing/announcements.json (hunk 2 above)
Comment (hannahbergam) [reply dashboard/config/marketing/announcements.json]: Added to the student page!

**Pull request merged**